docs/Development-Guides/Providers.md

```diff
@@ -200,3 +205,17 @@ Xero | `xero` | <span style="color:goldenrod">Assisted</span> | <span style="col
 Zoho Payroll | `zoho_payroll` | <span style="color:goldenrod">Assisted</span> | <span style="color:goldenrod">Assisted</span>
 
 **these assisted providers are **Pay Enabled**, meaning initial connection time is 14 days instead of 6 weeks*
+
+<!--
+type: tab
+title: ATS
+-->
+<!-- theme: info -->
+> Assisted Connect is not currently available for ATS integrations.
+
+### API Providers
+Display Name | Id |
+---------|----------
+Lever (beta) | `lever`
```

Comment:
Should we put `Automated` here?

It looks better than leaving it blank and then saying that Assisted isn't supported.